code/modules/economy/Accounts_DB.dm

```diff
@@ -19,7 +19,7 @@
 			return 0
 		if(access_cent_captain in held_card.access)
 			return 2
-		else if(access_hop in held_card.access || access_captain in held_card.access)
```

Comment:
Why is this an issue? It's an OR, so it should still allow the HoP to pass?

Reply:
No idea, it is not really needed, because the captain will have hop access, and changing this line fixes the issue with the hop being locked out of the finance console.

Reply:
How the fuck.

Reply:
It just works.

Reply:
OKAY.

Fix it to be this:
`else if((access_hop in held_card.access) || (access_captain in held_card.access))`